index.html

```diff
@@ -39,8 +43,9 @@ <h2>You´re in good company</h2>
       <img src="./images/ticketmaster-logo.png" alt="">
     </div>
     <div>
-      <p>Try it for free</p>
-      <p>Already using Slack? <a href="">Sign in</a></p>
+      <h2 class="text-center margin-small">Try it for free</h2>
```

Comment:
try to name your classes relevant to the elements function and not to the styles